addons/udes_common/models/ir_module.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, api
+
+
+class Module(models.Model):
+    _inherit = "ir.module.module"
+
+    @api.model
+    def get_module_installed(self, module_name):
+        """
+        Returns true if the supplied module exists 
+        and is installed otherwise false.
```

Comment:
Super nit, feel free to ignore! Can you add a comma after is installed, I read it as is installed otherwise the first 2 times and just got confused about the false.

Reply:
Always happy to make one-character changes! :smile: Will amend. :)

Reply:
Comma added. :)